server/routes/api/choropleth.py

```diff
@@ -35,15 +35,51 @@
   url_prefix='/tools/choropleth'
 )
 
-@bp.route('/api')
-def choropleth_api():
+@bp.route('/values')
+def choropleth_values():
     """Returns data for geographic subregions for a certain statistical 
             variable.
 
-    API Params: #TODO(iancostello): Split API calls up or move to client.
-        statVar -> The statistical variable to download, as a string.
-        perCapita -> Whether to return the per-capita value, as a string of a 
-                boolean.
+    API Params:
+        geoId -> The currently viewed geography to render, as a string.
+        level -> The subgeographic level to pull and display information for,
+                as a string. Choices: Country, AdministrativeLevel[1/2], City.
+        statVar -> The statistical variable to pull data about.
+
+    API Returns:
+        values -> statistical variable values for all subgeos.
+    """
+    # Get required request parameters.
+    requested_geoId = request.args.get("geoId")
+    if requested_geoId is None:
```

Comment:
The typical Python idiom is
```
if not requested_geoId:
```
This also catches the empty string.

Reply:
Changed